modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AspNetCoreServerCodegen.java

```diff
@@ -196,12 +314,14 @@ public void processOpts() {
                     packageFolder + File.separator + "Filters", "GeneratePathParamsValidationFilter.cs"));
         }
 
-        supportingFiles.add(new SupportingFile("wwwroot" + File.separator + "README.md", packageFolder + File.separator + "wwwroot", "README.md"));
-        supportingFiles.add(new SupportingFile("wwwroot" + File.separator + "index.html", packageFolder + File.separator + "wwwroot", "index.html"));
-        supportingFiles.add(new SupportingFile("wwwroot" + File.separator + "web.config", packageFolder + File.separator + "wwwroot", "web.config"));
+        if (generateWWWRoot) {
```

Comment:
My company doesn't use this folder, so I don't really know all of it's use cases. However, is there ever a case when someone would want to generate this with library?

Is it worth a separate flag or should it just be under the `isLib` flag?

Reply:
Its a good thought - we don't use it either I can move that to under isLibary and remove the additional flag.

Reply:
No comments from anyone so I have removed the option and used the buildTarget flag.